DynamicPackagingVODContent/StreamHLSAndDASH/src/main/java/sample/StreamHLSAndDASH.java

```diff
@@ -88,31 +80,47 @@ private static void runStreamHlsAndDash(ConfigWrapper config) {
 
         Scanner scanner = new Scanner(System.in);
         try {
-            // Ensure that you have customized encoding Transform with builtin preset. This is really a one time
+
+            MediaService account = manager.mediaservices().getByResourceGroup(config.getResourceGroup(),
+                    config.getAccountName());
+
+            account.update().withStorageAuthentication(null).apply();
+
+            // Ensure that you have customized encoding Transform with builtin preset. This
+            // is really a one time
             // setup operation.
+
             Transform transform = ensureTransformExists(manager, config.getResourceGroup(),
```

Comment:
Lately I have just been calling Transform CreateOrUpdate and not worrying about checking if it exists ever. If it does exist, this will just be an overwrite in this sample. Makes it easier to understand.